### PR TITLE
Post-reindex hotfixes

### PR DESCRIPTION
**Pull request by Andrew7234**
This branch has been pushed to dockerhub with the image tag `v0.4.0-andy`, and is currently running in production testnet/mainnet. ~~It is a live branch and will be updated as necessary.~~ Ready for review

### CODE REVIEW

File: storage/migrations/00_consensus.up.sql [hunk 1]
```diff
@@ -375,7 +375,7 @@ CREATE TABLE history.escrow_events
   epoch UINT63 NOT NULL,
   type TEXT NOT NULL,
   delegatee oasis_addr NOT NULL,
-  delegator oasis_addr NOT NULL,
+  delegator oasis_addr NOT NULL, -- NULL in 06_escrow_history_delegator.up.sql
```
Comment (pro-wh) [inline storage/migrations/00_consensus.up.sql:378]: what's the case where it's null?

**Review by pro-wh — APPROVED** [review]
ok

File: storage/migrations/00_consensus.up.sql (hunk 1 above)
Comment (Andrew7234) [reply storage/migrations/00_consensus.up.sql:378]: slashing events

Comment (pro-wh) [reply storage/migrations/00_consensus.up.sql:378]: oh ok

**Pull request merged**